oci-manifests/csi/openshift/machineconfig-csi.yml

```diff
@@ -0,0 +1,37 @@
+
+# 99_openshift-machineconfig_00-master-iscsi-service.yaml
```

Comment:
we can do it latter, but I think we can club these manifests with the CCM manifests altogether. I would reduce the amount of files to upload for the user.